public/data/Rahulbaran.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Rahul Kumar",
+  "type": "personal",
+  "bio": "Web developer | Love working with CSS, JavaScript and Flask",
```

Comment:
```suggestion
  "bio": "Web developer | Love working with CSS, JavaScript, and Flask",
```

Reply:
Should I delete this PR and make another one after removing the **comma**???

Reply:
You can push to the same branch and it will work!